_posts/2016-10-14-sarah-vang.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Not Enough"
+category: ourstories
+tags: 
+author: Sarah Vang
+featured-image: '/static/images/featured/2016-10-14-sarah-vang.jpg'
+featured-alt-text: "Side portrait of an Asian woman with photoshopped black smoke behind her. MYAAPIVOTE is for educational justice is at the bottom of the image."
+tweet: "Read YTEAMer Sarah Vang of @18millionrising share what her vote is for %23MyAAPIVote"
```

Comment:
ANY special characters (not letters or numbers) need URI encoding as well, such as @